script/jenkins/benchmark-agent-setup.sh

```diff
@@ -50,8 +51,8 @@ chmod 1777 /mnt/ramdisk
 
 # setup nat
 route delete default
-route add default gw 10.92.0.4
-route add -net 128.2.0.0/16 gw 10.92.0.1
+route add default gw 10.111.0.4
```

Comment:
I'm assuming this is internal stuff that is needed?

Reply:
Yes, it's actually unrelated to this PR but is an intended commit anyway.  I think I had it in place from a previous branch.